velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -1168,6 +1200,40 @@ TypePtr randMapType(
       randType(rng, scalarTypes, 0), randType(rng, scalarTypes, maxDepth - 1));
 }
 
+TypePtr randTypeByWidth(
+    FuzzerGenerator& rng,
+    const std::vector<TypePtr>& scalarTypes,
+    int minWidth) {
```

Comment:
minWidth is min or max width? Thanks!

Reply:
It's the lower bound of the width of the result type.